recipes/boost-ext-ut/all/conanfile.py

```diff
@@ -69,8 +69,7 @@ def package_id(self):
 
     def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "ut")
-        self.cpp_info.set_property("cmake_target_name", "boost")
-        self.cpp_info.components["ut"].set_property("cmake_target_name", "ut")
+        self.cpp_info.set_property("cmake_target_name", "boost::ut")
```

Comment:
```suggestion
        self.cpp_info.components["ut"].set_property("cmake_target_name", "boost::ut")
```
Shouldn't the property be set on the component ?

Reply:
I guess the original intention was to just have a target that is `boost::ut`, and the component was just a workaround for that as @SpaceIm said, so now that we can set target names freely we don't need components any more?

Reply:
exactly, now it will be interesting to see what happen in CMakeDeps with both set_property() and an unwanted component (due to .names backward support) living together.

Reply:
ok, sorry, I thought we were in the boost recipe !